app/models/source.rb

```diff
@@ -1,5 +1,5 @@
 class Source < ActiveRecord::Base
-  belongs_to :source_set
+  belongs_to :source_set, touch: true
```

Comment:
For reference on `touch: true` option, see http://guides.rubyonrails.org/association_basics.html#belongs-to-association-reference, section 4.1.2.8